tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Helpers/CombinatorialOrPairwiseDataAttribute.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="CombinatorialOrPairwiseDataAttribute.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.Helpers
+{
+    public class CombinatorialOrPairwiseDataAttribute : DataAttribute
+    {
+        public static bool UseFullTestConfiguration()
+        {
+            // TODO: consider interaction with RequiresThoroughTesting()
```

Comment:
Yeah, my inclination is to lean on `RequiresThoroughTesting()` instead, though it risks making PRs longer in that case when we wouldn't necessarily want it to 🤔

Reply:
Swapped to being set by that, maybe in the future we could make it a bit more configurable, but that just seems like additional complexity at the moment.